validation/src/main/scala/hmda/validation/rules/lar/macro/Q061.scala

```diff
@@ -20,14 +20,14 @@ object Q061 extends AggregateEditCheck[LoanApplicationRegisterSource, LoanApplic
 
   override def apply[as: AS, mat: MAT, ec: EC](lars: LoanApplicationRegisterSource): Future[Result] = {
 
-    val firstLienHoepaLoans =
-      count(lars.filter(lar => lar.hoepaStatus == 1 && lar.actionTakenType == 1 && lar.lienStatus == 1 && lar.rateSpread != "NA")
+    val firstLienPropertyLoans =
+      count(lars.filter(lar => lar.loan.propertyType == 1 && lar.actionTakenType == 1 && lar.lienStatus == 1 && lar.rateSpread != "NA")
         .filter(lar => lar.rateSpread.toInt >= 5))
```

Comment:
(I know this isn't the stated issue, but it's worth looking into while we're here) Should this rateSpread comparison really use `toInt`? Will that cause rounding issues?